package.json

```diff
@@ -1,23 +1,43 @@
 {
-  "name": "near-storage-bridge",
+  "name": "@textile/near-storage-bridge",
   "version": "1.0.0",
   "license": "(MIT AND Apache-2.0)",
+  "private": true,
   "scripts": {
-    "build": "asb",
-    "build:debug": "asb --target debug",
-    "deploy": "npm run build && npm run deploy:testnet && npm run deploy:edge",
-    "deploy:testnet": "near deploy filecoin-bridge.testnet",
-    "deploy:edge": "near deploy filecoin-bridge-edge.testnet",
-    "test": "npm run build:debug && asp"
+    "build": "npm run build --workspaces",
+    "build:debug": "npm run build:debug --workspaces",
+    "deploy": "npm run deploy --workspaces",
+    "deploy:testnet": "npm run deploy:testnet --workspaces",
+    "test": "npm run test --workspaces"
   },
+  "workspaces": [
```

Comment:
We're using workspaces to manage things. not the full blown npm module stuff, but really just making it easy to manage the three contracts and test, deploy them etc

Reply:
Do you plan on moving this into the same repo as eth?

Reply:
naw, they are just way too different. but I had toyed with the idea of moving the clients into the contract repos?